lib_nn/src/cpp/AggregateFn_DW.cpp

```diff
@@ -0,0 +1,193 @@
+#include <limits>
+
+#include "AggregateFn.hpp"
+#include "vpu_sim.h"
+
+using namespace nn;
+// TODO: [astew] CHAR_BIT not defined if I build with Cygwin or gcc+Ubuntu+WSL.
+// Not in <limits> either. [asj] this should be in limits.h
+#ifndef CHAR_BIT
```

Comment:
I couldn't this being used. Maybe it is used elsewhere? Otherwise, we can remove it.